Gemfile

```diff
@@ -23,7 +23,7 @@ group :test do
   gem "nokogiri", "~> 1.7"
   gem "rspec"
   gem "rspec-mocks"
-  gem "rubocop", "~> 1.8.1"
```

Comment:
@DirtyF

Reply:
Hello @DirtyF 
Im new Hear And token Tô signature digital Make this

Im sorry, ☺️